packages/fluent/scss/menu/_variables.scss

```diff
@@ -3,169 +3,196 @@
+$kendo-menu-border: initial !default;
+/// The shadow of the Menu.
 /// @group menu
-$kendo-menu-separator-margin: k-spacing(1) !default;
+$kendo-menu-shadow: none !default;
 
-/// Separator width
-/// @group menu
-$kendo-menu-separator-width: 1px !default;
 
-/// The base outline of focused menu item.
-/// @group menu
-$kendo-menu-item-focus-outline-offset: calc( #{k-spacing(0.5)} * -1 ) !default;
-$kendo-menu-item-focus-outline-width: 1px !default;
-$kendo-menu-item-focus-outline-style: solid !default;
+// Menu item
 
-/// The text color of the menu.
+/// The horizontal padding of Menu item.
 /// @group menu
-$kendo-menu-text: var( --kendo-body-text, inherit ) !default;
-/// The background color of the menu.
+$kendo-menu-item-padding-x: k-spacing(2) !default;
+/// The vertical padding of the Menu item.
 /// @group menu
-$kendo-menu-bg: transparent !default;
-/// The border color of the menu.
+$kendo-menu-item-padding-y: k-spacing(3) !default;
+
+/// The spacing between the Menu items.
 /// @group menu
-$kendo-menu-border: initial !default;
-/// The shadow of the menu.
+$kendo-menu-item-spacing: k-spacing(0) !default;
+/// The spacing between the Menu item text and icons.
 /// @group menu
-$kendo-menu-shadow: none !default;
+$kendo-menu-item-icon-spacing: var( --kendo-icon-spacing, .5rem ) !default;
+
+/// The font weight of the selected Menu item.
+/// @group menu
+$kendo-menu-item-selected-font-weight: var( --kendo-font-weight-bold, normal ) !default;
 
-/// The text color of menu items.
+/// The text color of Menu item.
 /// @group menu
 $kendo-menu-item-text: $kendo-menu-text !default;
-/// The background color of menu items.
+/// The background color of Menu item.
 /// @group menu
 $kendo-menu-item-bg: transparent !default;
 
-/// The text color of hover menu items.
+/// The text color of hovered Menu item.
 /// @group menu
 $kendo-menu-item-hover-text: var( --kendo-hover-text, inherit ) !default;
-/// The background color of hover menu items.
+/// The background color of hovered Menu item.
 /// @group menu
 $kendo-menu-item-hover-bg: var( --kendo-hover-bg, initial ) !default;
 
-/// The text color of focused menu items.
+/// The text color of active Menu item.
+/// @group menu
+$kendo-menu-item-active-text: $kendo-menu-item-hover-text !default;
+/// The background color of active Menu item.
+/// @group menu
+$kendo-menu-item-active-bg: var( --kendo-selected-bg, initial ) !default;
+
+/// The text color of focused Menu item.
 /// @group menu
 $kendo-menu-item-focus-text: $kendo-menu-item-text !default;
-/// The background color of focused menu items.
+/// The background color of focused Menu item.
 /// @group menu
 $kendo-menu-item-focus-bg: $kendo-menu-item-bg !default;
-/// The outline of focused menu items.
+
+/// The outline of focused Menu item.
 /// @group menu
 $kendo-menu-item-focus-outline: var( --kendo-focus-outline, none ) !default;
+$kendo-menu-item-focus-outline-offset: calc( #{k-spacing(0.5)} * -1 ) !default;
```

Comment:
This (and the following 2) variable is not documented

Reply:
I checked in other places where we document only the first (global) variable. For example in the Button: https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/button/_variables.scss#L224-L234

When we can document only the default variable and when to document them all?